raiden/network/proxies/token_network.py

```diff
@@ -1111,5 +1111,6 @@ def _check_channel_state_for_settle(self, participant1, participant2, channel_id
                 )
 
             raise RaidenRecoverableError(
-                'Channel cannot be settled before closing',
+                "Settling this channel failed although the channel's current state "
+                "is closed. Maybe it was already settled by the other participant?"
```

Comment:
This broke master, `make lint` doesn't pass because of missing comma :/

Reply:
Yeah this comma thing starts to look idiotic to me